h2o-core/src/test/resources/META-INF/services/water.H2OListenerExtension

```diff
@@ -0,0 +1 @@
+water.SimpleEverExpandingH2OListener
```

Comment:
how about NodeLocalEventCollectingListener?

my point is to make it obvious that this is not distributed & it collects events